assets/aircraft/js32.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "British Aerospace BAe-3200 Jetstream Super 31",
+    "icao": "JS32",
+    "engines": {
+        "number": 2,
+        "type": "S"
+    },
+    "weightClass": "M",
+    "category": {
+        "srs": 3,
+        "lahso": 5,
+        "recat": "F"
+    },
+    "ceiling": 25000,
+    "rate": {
+        "climb":      2000,
```

Comment:
For climb and descent rate, which "phase" from [eurocontrol](https://contentzone.eurocontrol.int/aircraftperformance/details.aspx?ICAO=JS32&ICAOFilter=js32) should I use? Right now I've used "initial climb" and "descent to FL100", but I don't know if that's correct.